src/RecipeTable.js

```diff
@@ -27,7 +26,12 @@ class IsRecipe extends Component {
                 <td>{recipe.malt}</td>
                 <td>{recipe.creator}</td>
                 <td>
-                  <button className="button smallButton">Edit</button>
+                  <button className="button smallButton" onClick={() => this.props.edit(
+                    recipe.name,
+                    recipe.ingredientType,
+                    recipe.malt,
+                    recipe.creator,
+                    recipe.id)}>Edit</button>
                   <button className="button smallButton" onClick={() => this.props.delete(recipe.name)}>Delete</button>
```

Comment:
You should create a method in this classe to replace `() => this.props.delete(recipe.name)`

Reply:
in ```() => this.props.delete(recipe.name)``` I am calling the ```Delete``` method from component Home. If I create a method for this matter wouldn't it do the same thing ?